### PR TITLE
[aptos-debugger] Added the bcs-txn-decoder

### PR DESCRIPTION
**Pull request by junkil-park**
### Description
* Added the bcs-exn-decoder to the aptos-debugger
* Added `README.md` to explain how to use it

Resolves https://github.com/aptos-labs/aptos-core/issues/8167.

### Test Plan
`cargo run -p aptos-debugger --bin bcs-txn-decoder`

### CODE REVIEW

**Review by runtian-zhou — COMMENTED** [review]
Was wondering if we should just move the reading part as an argument to the original debugger so that we don't need two separate binaries?

**Review by gerben-stavenga — APPROVED** [review]

File: aptos-move/aptos-vm/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/aptos-labs/aptos-core/edit/main/language/mov
 # MoveVM Runtime
 
 The MoveVM runtime is the verification and execution engine for the Move
-bytecode format. The runtime is imported and loaded in 2 modes:
+bytecode format. The runtime is imported and loaded in 2 mod0es:
```
Comment (rahxephon89) [inline aptos-move/aptos-vm/README.md:10]: Is this a typo?

Comment (junkil-park) [reply aptos-move/aptos-vm/README.md]: Oops, it's fixed now. Thanks!

**Review by rahxephon89 — APPROVED** [review]

**Comment by junkil-park** [comment]
> Was wondering if we should just move the reading part as an argument to the original debugger so that we don't need two separate binaries?

Right, I tried that initially, but it didn't nicely fit into the current CLI of `aptos-debugger`. Could this PR go as it is, and be merged into the `aptos-debugger` later when we refactor the CLI?

**Pull request merged**